server/blogRouter.mjs

```diff
@@ -0,0 +1,27 @@
+import express from "express";
+
+import * as db from "./db.mjs";
+
+const blogRouter = express.Router();
+
+blogRouter.get("/", async (request, response) => {
+  const posts = await db.getPostsWithComments();
+  response.json(posts);
+});
+blogRouter.get("/:id", async (request, response) => {
+  const post = await db.getPostById(request.params.id);
+  response.json(post);
+});
+
+blogRouter.use(express.json());
+blogRouter.post("/", async (request, response) => {
+  response.status(201).json(await db.addPost(request.body));
+});
+blogRouter.post("/:id", async (request, response) => {
```

Comment:
I feel like this should be a POST /posts/:id/comments.    doing a POST /posts/:id makes me feel like I'm editing something about the post it self.

Reply:
I trust your feeling & I'd love to talk more about this. Should I add /comments just for descriptive purposes?